### PR TITLE
Replace SIG/WG meeting placeholder references with Zoom links-1

### LINKED ISSUE #4588: Replace SIG/WG meeting placeholder references with Zoom links
Some time ago Zoom did not have sufficient moderation capabilities and the Zoom meeting links were removed from [sigs.yaml] and pointed towards this placeholder:
https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit

As a workaround to help mitigate trolls from causing problems in meetings. This has since been resolved and the entries in [sigs.yaml] should be updated with their correct link.

There are 53 references to the above placeholder as of 2020-03-09. Updated zoom links can be taken from either the SIG/WG's agenda or their calendar event from the [community calendar](https://calendar.google.com/calendar/embed?src=cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com&ctz=America/Los_Angeles).

/area community-management
/sig contributor-experience
/help
/good-first-issue

[sigs.yaml]: https://github.com/kubernetes/community/blob/master/sigs.yaml

### PR DESCRIPTION
**Pull request by AGMETEOR**
**Which issue(s) this PR fixes**:
Fixes #4588

### CODE REVIEW

**Comment by AGMETEOR** [comment]
Some notes
--------
* Regular Sig Meeting - Zoom link for time “7pm UTC/ 12:00pm PST ” is not available in the calendar

* Code organization office hours - Zoom link for time “2pm PST” is not available in the calendar

File: sigs.yaml [hunk 1]
```diff
@@ -260,23 +260,23 @@ sigs:
     time: "19:00"
     tz: UTC
     frequency: biweekly
-    url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
+    url: no-zoomlink-match
```
Comment (mrbobbytables) [inline sigs.yaml]: On these, I'd just leave it as the placeholder or reach out to the sig about updating it with the right time 👍

Comment (AGMETEOR) [reply sigs.yaml]: Let me work on that

Comment (AGMETEOR) [reply sigs.yaml]: @mrbobbytables reached out to the sig, got the links and I have now updated the PR.

**Comment by AGMETEOR** [comment]
Hello @mrbobbytables any more changes needed before this gets merged?

**Comment by mrbobbytables** [comment]
Oh sorry, missed the update for this 🤦‍♂ 
This is 100% my bad, but the I didn't catch that apps was also being updated 😬 
Could you change the commit message quick to:
```
Update sig-api-machinery, sig-apps and sig-architecture zoom meeting links
```

**Comment by mrbobbytables** [comment]
Thanks!
/lgtm
/approve

**Pull request merged**